frontend/src/app/rulesets-overview/rulesets-overview.component.html

```diff
@@ -1,10 +1,19 @@
-<div class="ruleset-card new" [routerLink]="['new']">
-    <mat-icon>add</mat-icon>
-</div>
-<div *ngFor="let ruleset of rulesets" class="ruleset-card" [routerLink]="[ruleset.rulesetId, 'edit']">
-    <h2 class="title">{{ ruleset.name }}</h2>
-    <div class="content">
-        <p class="description" ovideEllipsis>{{ ruleset.description }}</p>
-        <div class="meta-data">{{ ruleset.createdAt | date }}, {{ ruleset.createdBy }}</div>
+
+<div *ngIf="(rulesets$ | async) as rulesets; else loading" class="rulesets-container" [@cardInAnimation]>
```

Comment:
This produces an infinite loading spinner when there is an error while retrieving the rulesets. This should be handled by #98 in the future.